### PR TITLE
[DefaultWebCertificateProvider] Translate alert view messages

### PR DESCRIPTION
**Pull request by VincentDondain**

### CODE REVIEW

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide/DefaultWebCertificateProvider.cs [hunk 1]
```diff
@@ -48,9 +48,9 @@ public bool GetIsCertificateTrusted (string uri, string certificateFingerprint)
 				using (var handle = new System.Threading.ManualResetEvent (false)) {
 					Gtk.Application.Invoke (delegate {
 						value = MessageService.AskQuestion (
-							"Untrusted HTTP certificate detected",
-							string.Format ("Do you want to temporarily trust this certificate in order to" +
-							" connect to the server at {0}?", uri),
+							GettextCatalog.GetString ("Untrusted HTTP certificate detected"),
+							GettextCatalog.GetString (string.Format ("Do you want to temporarily trust this certificate in order to" +
```
Comment (Therzok) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide/DefaultWebCertificateProvider.cs]: Remove the string.Format call.

Comment (VincentDondain) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide/DefaultWebCertificateProvider.cs]: Why?

Comment (Therzok) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide/DefaultWebCertificateProvider.cs]: Checked via the Regex Toolkit, we fail to parse this kind of string.

regex

Comment (VincentDondain) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide/DefaultWebCertificateProvider.cs]: Ah I see I see

Comment (Therzok) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide/DefaultWebCertificateProvider.cs]: Regex is from here: https://github.com/mono/monodevelop/blob/master/main/src/addins/MonoDevelop.Gettext/MonoDevelop.Gettext.addin.xml#L144-L146

Comment (VincentDondain) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide/DefaultWebCertificateProvider.cs]: Yes thanks I found it (:

That will be useful in the future.

**Pull request merged**